tests/MongoDB.Bson.Tests/MongoDB.Bson.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp2.1;net452</TargetFrameworks>
+    <TargetFrameworks>netcoreapp1.0;netcoreapp2.0;net452</TargetFrameworks>
```

Comment:
I'd recommend targeting the LTS/ex-lts versions of .NET Core since those are the ones that are most likely still in use in the wild. So netcoreapp2.1 and netcoreapp1.1

(Based off https://dotnet.microsoft.com/platform/support/policy/dotnet-core and https://csharp.christiannagel.com/2018/06/26/ltsandcurrent/ )